mms/metrics_manager.py

```diff
@@ -57,39 +72,64 @@ def start(metrics_write_to, namespace, mutex):
+                                                                model_name=model_name,
+                                                                unit=Unit['count'],
+                                                                aggregate_method='interval_sum',
+                                                                write_to=metrics_write_to)
+            MetricsManager.metrics['request_metric'] = Metric('requests', mutex,
+                                                              model_name=model_name,
+                                                              unit=Unit['count'],
+                                                              aggregate_method='interval_sum',
+                                                              write_to=metrics_write_to)
+            MetricsManager.metrics['overall_latency_metric'] = Metric('overall_latency', mutex,
+                                                                      model_name=model_name,
+                                                                      unit=Unit['ms'],
+                                                                      aggregate_method='interval_average',
+                                                                      write_to=metrics_write_to)
+            MetricsManager.metrics['inference_latency_metric'] = Metric('inference_latency', mutex,
+                                                                        model_name=model_name,
+                                                                        unit=Unit['ms'],
+                                                                        aggregate_method='interval_average',
+                                                                        write_to=metrics_write_to)
+            MetricsManager.metrics['pre_latency_metric'] = Metric('preprocess_latency', mutex,
+                                                                  model_name=model_name,
+                                                                  unit=Unit['ms'],
+                                                                  aggregate_method='interval_average',
+                                                                  write_to=metrics_write_to)
+
         MetricsManager.metrics['cpu_metric'] = Metric('cpu', mutex,
-                                                      namespace=namespace,
+                                                      model_name=None,
+                                                      unit=Unit['percent'],
+                                                      is_resource=True,
                                                       aggregate_method='interval_average',
                                                       write_to=metrics_write_to,
                                                       update_func=cpu)
         MetricsManager.metrics['memory_metric'] = Metric('memory', mutex,
```

Comment:
Is this process memory, or host memory? Can we clarify this with proper naming? e.g. host_memory or process_memory
Same applies to cpu and IO.

Reply:
changed.